packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -1,22 +1,37 @@
 <template>
 	<div class="search-bar-container">
-		<span class="p-input-icon-left p-input-icon-right">
-			<i class="pi pi-search" />
-			<InputText
-				type="text"
-				class="input-text"
```

Comment:
@YohannParis I just noticed I got rid of this class when I was merging main, but I see you added it in a previous commit. Do we need this since we can just use `.p-inputtext` for styling?

Reply:
I added it back because I don't want our scope styling to be dependant of a class we do not control. This `p-inputtext` might change or be removed with an update from PrimeVue.